2 - cluster organisations and link them to the STM topics/clustering.ipynb

```diff
@@ -5,20 +5,22 @@
    "id": "b92f4c59-1683-44d3-a57c-02f07c61027b",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #32.        Y = tsne.fit_transform(np.array(vectors))
</pre><p><code>vectors</code> is here still a list, so it needs to be converted for t-SNE (requires <code>.shape</code> attribute).  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/disaster-capitalism/named-entities/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
I don't follow here. np.array converts to the correct format no? ``.shape`` is to see the dimensions of the data right? Or maybe I misunderstand something.

Reply:
<p>What I meant is that <code>vectors</code> is initiated as a list, so it throws an error when passed as an argument to <code>tsne.fit_transform()</code>. To avoid that, it should be converted to a numpy array first.   </p>

Reply:
Ah sorry my bad... I see that it was you who suggested that line of code including "np.array()". I initially thought I already wrote that code and then I was wondering: "its already there, what is Malte talking about?" XD I see now yes that it is a list and not a numpy array. But... what is super weird is that it does not throw an error for me. It runs fine with list input for me... :/ and the clusters make sense. I wonder if I implement your change if it will affect the clustering behaviour or if I use a different version of sklearn which automatically converts list input to np array. I am using version 1.1.0 for sklearn.

Reply:
Ah, yeah sorry, I understand why that is confusing. Maybe I should have made that more clear (that this was my suggestion). My intuition says it should not affect the clustering output and that it probably converts lists to arrays under the hood anywaw. But we can also leave it as it is just to be sure.